dag_in_context/src/schedule.rs

```diff
@@ -104,19 +115,29 @@ pub fn mk_sequential_schedule() -> Vec<String> {
     res
 }
 
-/// Parallel schedule must return a single string,
-/// a schedule that runs optimizations over the egraph.
 pub fn parallel_schedule() -> Vec<String> {
     let helpers = helpers();
 
-    vec![format!(
-        "
+    vec![
+        format!(
+            "
 (run-schedule
```

Comment:
This is currently doing
```
vec![
passthrough,
loop-inversion,
main-rulesets
]
```
After changing to 
```
vec![
loop-inversion,
passthrough,
main-rulesets
]
```
It does not terminate, interesetingly

Reply:
What benchmark doesn't terminate? It might be related to dead code, the thing I mentioned last meeting